dnsmasq/run.sh

```diff
@@ -25,6 +38,51 @@ for host in $(bashio::config 'hosts|keys'); do
     echo "address=/${HOST}/${IP}" >> "${CONFIG}"
 done
 
+# Create DHCP config
+if ! bashio::config.is_empty 'dhcp.lease_time';
+then
+    LEASE_TIME=$(bashio::config 'dhcp.lease_time')
+else
```

Comment:
This can be re-written without the else. Move the contents of the else right above the if statement.

Reply:
fixed in the recent commit